### DISCUSSION TITLE
FlySky gimbal on Radiomaster

### DISCUSSION THREAD

**Discussion by robb1364** [EdgeTX/edgetx#993] [Q&A] [answered]
Please can anyone send me update link for new Edge release firmware to anable Flysky gimball?
Thanks Robb

**Comment by pfeerick** [comment, marked as answer]
For what transmitter? TX16S? You can get a build of 2.5.0 and also the latest nightly from here : http://files.portabledev.xyz/d/8e4d08b0905743cbbccb/

**Comment by robb1364** [comment]
Hello Peter, thank you very much for your super fast reply and help! Everythink is working perfectly now.
